dipy/reconst/tests/test_shm.py

```diff
@@ -377,6 +449,24 @@ def test_sf_to_sh():
     odf2 = sh_to_sf(odf_sh, sphere, 8, "descoteaux07")
     assert_array_almost_equal(odf, odf2, 2)
 
+    # Try out full bases with order 7 (64 coefficients) instead of 8
+    # (exactly 81 coefficients)
+    odf_sh = sf_to_sh(odf, sphere, 7, "tournier07_full")
+    odf2 = sh_to_sf(odf_sh, sphere, 7, "tournier07_full")
+    assert_array_almost_equal(odf, odf2, 2)
+
+    odf_sh = sf_to_sh(odf, sphere, 7, "descoteaux07_full")
```

Comment:
Are you still using a ``hemi_icosahedron`` here? I think you should use a full sphere as this is a full (non-symmetric) basis.

Reply:
Yes, the test uses a `hemi_icosahedron` for the full basis. I'll replace it by a full sphere.